cmake/BLTMacros.cmake

```diff
@@ -77,76 +77,74 @@ endmacro(blt_list_remove_duplicates)
 
 
 ##------------------------------------------------------------------------------
-## blt_add_target_definitions(TO <target> TARGET_DEFINITIONS [FOO [BAR ...]])
+## blt_add_target_definitions(TO <target>
+##                            SCOPE <public (default)|private|interface>
```

Comment:
Do we have a way to indicate optional vs. required parameters?
Similarly for ``options`` vs. ``singleValueArgs`` vs. ``multiValueArgs``?